### PR TITLE
Move generic code from R-Car machine

### PR DESCRIPTION
**Pull request by iartemenko**
Add meta layer 'meta-xt-domx' as a storage
for machine and platform unspecific recipes.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb [hunk 1]
```diff
@@ -10,6 +10,14 @@ S = "${WORKDIR}/repo"
 # path for machine specific layers and recipes
 FILESEXTRAPATHS_prepend := "${THISDIR}/../../machine:"
 
+# path for machine unspecific recipes
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: machine agnostic/non-specific

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb [hunk 2]
```diff
@@ -10,6 +10,14 @@ S = "${WORKDIR}/repo"
...
+FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
+
+XT_QUIRK_UNPACK_SRC_URI += "file://meta-xt-domx;subdir=repo"
+
+# these layers will be added to bblayers.conf on do_configure
+XT_QUIRK_BB_ADD_LAYER += " meta-xt-domx"
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: meta-xt-images-domx

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb (hunk 1 above)
Comment (iartemenko) [reply recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: Done

File: recipes-dom0/dom0-image-weston/dom0-image-weston.bb (hunk 2 above)
Comment (iartemenko) [reply recipes-dom0/dom0-image-weston/dom0-image-weston.bb]: Done

File: recipes-domu/domu-image-weston/domu-image-weston.bb [hunk 3]
```diff
@@ -24,5 +32,6 @@ SRCREV = "${AUTOREV}"
 # Renesas R-Car
 ################################################################################
 XT_QUIRK_UNPACK_SRC_URI_append_rcar = " file://meta-rcar-gen3;subdir=repo"
+
```
Comment (andr2000) [inline recipes-domu/domu-image-weston/domu-image-weston.bb]: Not relevant change

**Comment by andr2000** [comment]
Lgtm, rb

**Pull request merged**